src/partition/ReceivedPartition.cpp

```diff
@@ -104,7 +104,7 @@ void ReceivedPartition::compute()
 
   // check to prevent false configuration
   if (not utils::MasterSlave::isSlave()) {
-    PRECICE_CHECK(_fromMapping || _toMapping,
+    PRECICE_CHECK(not(_fromMappings.empty() && _toMappings.empty()),
```

Comment:
We could bundle this in `anyMapping()`, `hasNoMapping()` or `hasAnyMapping()`.
This would simplify things a bit here.